com.github.ahrm.sioyek.yaml

```diff
@@ -0,0 +1,75 @@
+---
+app-id: com.github.ahrm.sioyek
+
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: sioyek
+
+finish-args:
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --filesystem=/tmp
+  - --share=ipc
+  # Sioyek fails with wayland (invisible window)
+  #- --socket=wayland
+  #- --socket=fallback-x11
+  - --socket=x11
+  - --device=shm
+  - --device=dri
+
+modules:
+  - org.boost.Boost.json
```

Comment:
you didn't add the content of the manifest here.

Reply:
Hopefully now fixed.